binding/react-native/ios/Porcupine.swift

```diff
@@ -9,137 +9,164 @@
 // specific language governing permissions and limitations under the License.
 //
 
-import PvPorcupine
-import Foundation
-
-enum PvPorcupineError: Error {
-    case RuntimeError(_ message: String)
-}
+import Porcupine
 
 @objc(PvPorcupine)
 class PvPorcupine: NSObject {
-    private var porcupinePool:Dictionary<String, OpaquePointer?> = [:]
+    private var porcupinePool:Dictionary<String, Porcupine> = [:]
 
-    @objc func constantsToExport() -> Dictionary<String, Any> {        
-        
-        let modelPath : String = Bundle.main.path(forResource: "porcupine_params", ofType: "pv") ?? "unknown"
-        
-        let keywordPaths = Bundle.main.paths(forResourcesOfType: "ppn", inDirectory: nil)                        
-        var keywordDict:Dictionary<String, String> = [:]
-        for keywordPath in keywordPaths{
+    @objc(fromBuiltInKeywords:modelPath:keywords:sensitivities:resolver:rejecter:)
+    func fromBuiltInKeywords(accessKey: String, modelPath: String, keywords: [String], sensitivities: [Float32],
+        resolver resolve:RCTPromiseResolveBlock, rejecter reject:RCTPromiseRejectBlock) -> Void {
+        do {
```

Comment:
Can we have more shared code between `fromBuiltInKeywords` and `fromKeywordPaths`?